.gitignore

```diff
@@ -44,3 +44,4 @@ symbols.txt
 /CMakePresets.json
 /compile_commands.json
 /scratch
+awslcTestTmpFile*
```

Comment:
Thank you.....I also just want to track down what is putting these here and make it write to `$TEMP` at some point.